### PR TITLE
big prompts bundle.js

### PR DESCRIPTION
**Pull request by tioopental**
Added lots more prompts, small corrections

### CODE REVIEW

File: src/prompts/neuro.js [hunk 1]
```diff
@@ -27,7 +27,6 @@ export default [
   'backslash spam',
   'mentions Filipino boy',
   '1984',
-  'ignores bits message',
```
Comment (tioopental) [inline src/prompts/neuro.js:30]: same as donowalled

File: src/prompts/neuro.js [hunk 2]
```diff
@@ -47,9 +46,45 @@ export default [
...
+  'Technical Difficulties screen',
+  'Hype train lvl 5+',
+  'raiding anny or raided by anny',
+  'admits she\'s an AI',
+  'meaning of life',
+  'chat spams WAYTOODANK',
+  'Why?',
+  'world or humankind conquering plan',
+  'neuroWide',
+  'chat spams Vedal\'s catchphrase',
```
Comment (cjmaxik) [inline src/prompts/neuro.js:82]:
```suggestion
  'chat spams Vedal\'s copypasta',
```

Or maybe
```suggestion
  'this is crazy 🐢',
```
We don't have much space for the sentence.

File: src/prompts/neuro.js [hunk 3]
```diff
@@ -47,9 +46,45 @@ export default [
...
+  'wholesome-sama',
```
Comment (cjmaxik) [inline src/prompts/neuro.js:83]: What is this exactly?

File: src/prompts/neuro.js [hunk 4]
```diff
@@ -47,9 +46,45 @@ export default [
...
+  'apologises profusely',
+  'misgenders random person',
+  'chat spams Erm',
+  'rickroll-sama',
```
Comment (cjmaxik) [inline src/prompts/neuro.js:87]:
```suggestion
  'rickrolls',
```

File: src/prompts/neuro.js [hunk 5]
```diff
@@ -47,9 +46,45 @@ export default [
...
+  'chat spams GOODONE',
```
Comment (cjmaxik) [inline src/prompts/neuro.js:88]: We already have `funny joke`

File: src/prompts/neuro.js [hunk 6]
```diff
@@ -47,9 +46,45 @@ export default [
...
+  'admits she\'s an AI',
+  'meaning of life',
+  'chat spams WAYTOODANK',
+  'Why?',
+  'world or humankind conquering plan',
+  'neuroWide',
+  'chat spams Vedal\'s catchphrase',
+  'wholesome-sama',
+  'apologises profusely',
+  'misgenders random person',
```
Comment (cjmaxik) [inline src/prompts/neuro.js:85]:
```suggestion
  'misgenders someone',
```

File: src/prompts/neuro.js [hunk 7]
```diff
@@ -47,9 +46,45 @@ export default [
...
   'mentions favorite food',
   'wot? / what?',
   'claims to stream later',
-  'donowalled'
+  'donowalled',
+  'mentions turtles',
+  'chat spams AintNoWay',
+  'mentions random country',
+  'chat spams AYAYA',
+  'believe in the time god',
```
Comment (cjmaxik) [inline src/prompts/neuro.js:58]:
```suggestion
  'mentions the Time God',
```

File: src/prompts/neuro.js [hunk 8]
```diff
@@ -47,9 +46,45 @@ export default [
...
+  'KEKWa',
+  'Noted',
+  'chat spams Clueless',
+  'POG',
```
Comment (cjmaxik) [inline src/prompts/neuro.js:62]:
```suggestion
  'POG / poggers',
```

**Review by cjmaxik — CHANGES_REQUESTED** [review]

**Comment by tioopental** [comment]
Fixed

**Comment by cjmaxik** [comment]
Thank you. heart

**Pull request merged**